src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -79,4 +81,9 @@ export class MdTabLink {
     '[class.mat-tab-link]': 'true',
   },
 })
-export class MdTabLinkRipple extends MdRipple {}
+export class MdTabLinkRipple extends MdRipple {
+  constructor(elementRef: ElementRef, ngZone: NgZone, ruler: ViewportRuler,
+              @Optional() @Inject(MD_DISABLE_RIPPLES) forceDisableRipples: boolean) {
+    super(elementRef, ngZone, ruler, forceDisableRipples);
+  }
```

Comment:
An issue in core where the automatic constructor inheritance did not work because the classes span across different modules (Here, a class in `MdTabsModule` inheriting from `MdRippleModule`). In practice what was happening is that the elementRef was undefined in the ripple constructor, causing the component and app to fail.

This didn't show up in our demo app but was coming up in some Google apps.